addons/mrp/models/mrp_routing.py

```diff
@@ -104,6 +106,22 @@ def _compute_workorder_count(self):
         for operation in self:
             operation.workorder_count = count_data.get(operation.id, 0)
 
+    @api.depends('bom_id.product_id', 'bom_id.product_qty', 'workcenter_id.time_start', 'workcenter_id.time_stop', 'workcenter_id.capacity_ids', 'time_cycle')
+    def _compute_cycle_number_total_time(self):
```

Comment:
it could be merge with _compute_time_cycle? Not sure to check if it's better or not

Reply:
merging the computes gives:

UserWarning: mrp.routing.workcenter: inconsistent 'compute_sudo' for computed fields
time_cycle, cycle_number, time_total, show_time_total.
Either set 'compute_sudo' to the same value on all those fields,
or use distinct compute methods for sudoed and non-sudoed fields.

merge / don't merge ?